tensorflow/python/ops/embedding_ops.py

```diff
@@ -322,6 +322,40 @@ def embedding_lookup(
   Raises:
     ValueError: If `params` is empty.
   """
+  
+  
+  """
+    **Behavior Difference between CPU and GPU**
+
+    Please note that when using `tf.nn.embedding_lookup` on a GPU, if an out-of-bound 
+    index is encountered, a value of 0 will be stored in the corresponding output value. 
```

Comment:
This is the case for all GPU kernels involving indexing.  It's still considered invalid, but it's not worth us trying to catch it (it hurts performance too much).

Reply:
Thank you for clarifying the behavior on GPU regarding out-of-bound indices and the decision not to actively catch these cases due to performance considerations. I appreciate your feedback and will make sure to retain the information in the code comments to ensure transparency and avoid confusion for anyone working with the code.